document_similarity/microservice/routes/service.py

```diff
@@ -92,18 +94,20 @@ def update_similarities():
             WHERE document_id={};
             """.format(document_id)
         document_text = (pg.execute(statement))[0]['fulltext_cleaned']
-        if document_text == "":
+        if document_text == "" or document_text is None:
             statement = """
             SELECT abstract FROM documents
             WHERE document_id={};
             """.format(document_id)
             document_text = (pg.execute(statement))[0]['abstract']
-            if document_text == "":
+            if document_text == "" or document_text is None:
```

Comment:
You can unnest those if statements and the code will still work and be more beatiful.
```
doc_test = something
if doc_test is None:
    doc_test = something
if doc_test is None:
    doc_test = something
if doc_test is None:
    raise Exception()
```